lib/symbioticpy/symbiotic/symbiotic.py

```diff
@@ -645,6 +645,11 @@ def _run_symbiotic(self):
         self.instrument()
 
         passes = self._tool.prepare()
+        # make all memory symbolic (if desired)
```

Comment:
I'm not sure why this (and the `-instrument-alloc` below) have been moved here from `tools/klee.py`? They are more or less KLEE specific. I think it can break code for other verifiers.